moonshot-readiness.py

```diff
@@ -28,61 +46,59 @@ def test_basic():
     global results
     print("\n\nTesting task basic...")
 
-
 #Hostname is FQDN
 
-    cmd = os.popen("hostname -f")
+    cmd = os.popen("%s -f" % bin_hostname)
```

Comment:
If you're constructing the command at runtime, you should format Popen like this:

```
os.popen([bin_hostname, '-f'])
```

For conciseness, and making sure Popen does the right thing with the arguments.